src/domains/events.ml

```diff
@@ -9,7 +9,7 @@ type t =
   | AssignSpawnedThread of lval * ThreadIdDomain.Thread.t (** Assign spawned thread's ID to lval. *)
 
 let pretty () = function
-  | Lock m -> dprintf "Lock %a" LockDomain.Addr.pretty m
+  | Lock m -> dprintf "Lock %a" LockDomain.Addr.pretty m (* This is only emitted if the mutex was not previously held *)
```

Comment:
This should be a documentation comment on the constructor in the type definition above, not in the `pretty` implementation.